scripts/gen_hashed_bin.py

```diff
@@ -74,7 +74,7 @@ def append_hashes(outf, in_fname):
 		elif len(page) == 0 :
 			break
 		else :
-			print "Error: short read, got " + repr(len(page))
+			print("Error: short read, got " + repr(len(page)))
```

Comment:
I wonder if we should put a"
```python
from __future__ import print_function
```
without that, it'll work without any arguments. With this line, you get the whole print function, complete with its keyword arguments.